src/templates/basicDoc.js

```diff
@@ -14,14 +15,20 @@ import toString from 'mdast-util-to-string';
 import DefaultRelatedContent from '../components/DefaultRelatedContent';
 
 const BasicDoc = ({ data }) => {
-  const { mdx } = data;
+  const { mdx, site } = data;
   const {
     mdxAST,
     frontmatter,
     body,
     fields: { fileRelativePath },
   } = mdx;
 
+  const issueUrl = `${site.siteMetadata.repository}/issues/new`;
+  const title = `Feedback+for:+${frontmatter.title.replace(' ', '+')}`;
+  const labels = `eng,feedback`;
+  const positiveFeedback = `${issueUrl}?labels=${labels},feedback-positive&title=${title}`;
+  const negativeFeedback = `${issueUrl}?labels=${labels},feedback-negative&title=${title}`;
```

Comment:
Will we ever have a case where the feedback might point to issues other than these? I wonder if maybe we should move this inside the `SimpleFeedback` component. That way if we need to include the feedback widget on other pages with right rails, we don't need to copy/paste this over. Thoughts?

Reply:
I'm down with that. We would then only need to pass in the page title (which could be optional).